### DISCUSSION TITLE
The new fork page should show my fork(s) of this repo if I've forked the repo already, like in the past

### DISCUSSION THREAD

**Discussion by mercury233** [community/community#14745] [Other Feature Feedback, Questions, & Ideas]
I'm been using that feature to jump to my fork

**Comment by nyngwang** [comment]
@mercury233: LOL, I thought it was just me who misclicked something. Could you verify that this is your problem?

Old (and good)

<img src="https://i.stack.imgur.com/zLMFT.png" width="700">

New (and tedious)

<img src="https://user-images.githubusercontent.com/24765272/162890252-d4254e6e-8b30-43c7-b91d-a4a396788cac.png" width="700">

Oh please make the old behaviour back!

**Reply by mercury233** [reply]
Yes, I mean the section "You've already forked" disappeared.

**Comment by unitycoder** [comment]
yes and please, i need 1 click fork, not interested to fill in useless details when forking.. (and can do that in old behavior, as it opens the forked repo and can edit description or go rename it, *personally never have had the need for that though)

**Reply by mercury233** [reply]
I guess github team agreed to https://github.com/github/feedback/discussions/14349 and added some confirm for fork.

**Reply by nyngwang** [reply]
@mercury233 I did notice that too

**Reply by nyngwang** [reply]
Why they don't provide a toggle so both parties will be happy...

**Comment by unitycoder** [comment]
Today discovered even worse feature on the new Fork feature,
it automatically assigns different organization as owner(!) if you already had forged it in your account.,

it just picks next org in the list (alphabetic order), cannot imagine who would ever want that to happen? (getting separate fork automatically on random repo, if you already have it forked). And its too easy to accidentally fork on that wrong org now, if you dont carefully check it.

**Comment by Jacke** [comment]
Why the hell did you have broke the fork window? It was nice. Now it's ugly as torture. Why?
